src/main/java/xyz/nucleoid/plasmid/game/GameWorld.java

```diff
@@ -153,10 +234,24 @@ public void tick() {
         this.invoker(GameTickListener.EVENT).onTick();
     }
 
+    /**
+     * Requests that this {@link GameWorld} begins, which invokes any {@link RequestStartListener#EVENT} listener attached to it.
```

Comment:
Maybe can also mention that the meaning of a game being "started" is up to interpretation of the game implementation and that it's specifically connected to `/game start`